usaspending_api/accounts/serializers.py

```diff
@@ -126,7 +126,6 @@ class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
 
 class AgenciesFinancialBalancesSerializer(serializers.Serializer):
 
-    fiscal_year = serializers.IntegerField()
```

Comment:
@nmonga91 Took another look at this--the `annotate` statement Sums the values does need the fiscal year to be specified, because that's the group by field. We could technically don't need to annotate `submission__reporting_fiscal_year` into `fiscal_year` for this to work, but I kind of like it that way for readability. What do you think?